### ISSUE TITLE
Purging CI Windows jobs

### ISSUE THREAD

**Issue by StefanStojanovic** [libuv/libuv#3945]
After reviewing Node.js Windows CI jobs, I've noticed there's a libuv job that's not used anymore: https://ci.nodejs.org/view/All/job/libuv-test-commit-windows/. Since we already have all jobs backed up at https://github.com/nodejs/jenkins-config-test, I'd like to remove that one from Jenkins. I plan to do this next Wednesday (April 12), so if someone thinks it should be kept, please let me know until then.

Refs: https://github.com/nodejs/build/issues/3285

**Comment by vtjnash** [comment]
Yes, that configuration hasn't been supported in a long time (removed in 53f3c687fc288708721a5a3d9563febda1b9d2c1). The current builder is https://ci.nodejs.org/view/All/job/libuv-test-commit-windows-cmake/

**Comment by StefanStojanovic** [comment]
Job deleted.
